src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event extends Task {
+    protected String start;
+    protected String end;
+
+    public Event(String description,  String start, String end) {
+        super(description);
+        this.start = start;
+        this.end = end;
+    }
+
+    public String getTypeIcon() {
+        return "E";
+    }
+
+    @Override
+    public String toString() {
+        return "[" + this.getTypeIcon() + "][" + this.getStatusIcon() + "] " + this.description + "(from: " + this.start + "to:" + this.end + ")";
```

Comment:
```suggestion
        return "[" + this.getTypeIcon() + "][" + this.getStatusIcon() + "] " 
        + this.description + "(from: " + this.start + "to:" + this.end + ")";
```
I think splitting the code into a newline would be useful 😃